### PR TITLE
chore: load the base64 in `utils`, different ENV use different way.

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

**Comment by membphis** [comment]
after this PR is merged, I'll make a try to release a new version. 

then we can start to use the latest version in Apache APISIX.

File: rockspec/skywalking-nginx-lua-master-0.rockspec
Comment (wu-sheng) [inline rockspec/skywalking-nginx-lua-master-0.rockspec:1]: What is this for? Provide one for dev version?

**Comment by wu-sheng** [comment]
@membphis Release still has been processed by at least a committer. @moonming do you plan to help about this?

**Comment by wu-sheng** [comment]
@membphis Your branch and fork repo are readonly, please update the branch by yourself.

**Comment by moonming** [comment]
yes, I can help to release the new version.

File: rockspec/skywalking-nginx-lua-master-0.rockspec
Comment (membphis) [reply rockspec/skywalking-nginx-lua-master-0.rockspec:1]: only for developers. the developer will put the latest configuration here. 
when we release a new version, then we can copy it from this file directly.

Apache APISIX uses this way too:

https://github.com/apache/apisix/blob/master/rockspec/apisix-master-0.rockspec

**Comment by membphis** [comment]
> @membphis Your branch and fork repo are readonly, please update the branch by yourself.

thx for your mind, I have merged the master branch.

**Comment by wu-sheng** [comment]
As you are planning to release after this PR, where is the 0.3.0 rockspec file?

**Comment by membphis** [comment]
> As you are planning to release after this PR, where is the 0.3.0 rockspec file?

after I have tested, I think we can release a new version.

**Review by wu-sheng — APPROVED** [review]
Please make the rockspec file exists before tagging. Anyway, tag is @moonming 's responsibility.

**Pull request merged**
